javascript/ql/lib/change-notes/2024-01-30-certification_not_certificate.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* The string "certification" isn't a certificate, and should be excluded from the maybeCertificate() regex.
```

Comment:
Maybe focus more on what the visible change is to end users?

Maybe something like: 

```
The name `certification` is no longer seen as possibly being a certificate, and will therefore no longer be flagged in queries like `clear-text-logging` that look for sensitive data.
```

Reply:
Updated to be more nearly like that text, please review at your convenience.